go.mod

```diff
@@ -26,6 +26,10 @@ require (
 
 require (
 	4d63.com/gochecknoglobals v0.1.0 // indirect
+	cloud.google.com/go v0.102.1 // indirect
```

Comment:
can you run a go mod tidy. not sure why these showed up

Reply:
Done